new-lamassu-admin/src/pages/Blacklist/Blacklist.js

```diff
@@ -108,6 +154,8 @@ const Blacklist = () => {
 
   const rejectAddressReuse = complianceConfig?.rejectAddressReuse ?? false
 
+  const enablePaperWalletOnly = complianceConfig?.enablePaperWalletOnly ?? false
```

Comment:
You can change line 155 as well, to simplify code

```suggestion
  const enablePaperWalletOnly = !!complianceConfig.enablePaperWalletOnly
```